src/classes/RLLP_OppPartialSoftCreditRollup.cls

```diff
@@ -161,7 +161,13 @@ public class RLLP_OppPartialSoftCreditRollup {
                 toUpdate.add(newC);
             }
         }
-        update toUpdate;
+        //update toUpdate;
+        List<Database.SaveResult> srList = Database.update(toUpdate, false);
+
+        ERR_Handler.Errors errorResult = ERR_Handler.getJobErrors(srList, toUpdate);
+        if (errorResult.errorsExist) {
+            ERR_Handler.processErrors(errorResult.errorRecords, 'Jobs');
```

Comment:
It's probably better to use the `ERR_Handler_API.Context.RLLP.name()` enum instead of the hardcoded values of `'Jobs'` for all the calls to processErrors()

Reply:
I agree. There was another place in the code that used a different convention, and I wasn't sure which was the better way to do it so ended up passing both over!